test/e2e/specs/site-editor/navigation-editor.spec.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Editing Navigation Menus', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllMenus();
+	} );
+
+	test( 'it should lock the root Navigation block in the editor', async ( {
+		admin,
+		page,
+		pageUtils,
+		requestUtils,
+		editor,
+	} ) => {
+		// We could Navigate directly to editing the Navigation Menu but we intentionally do not do this.
+		//
+		// Why? To provide coverage for a bug that caused the Navigation Editor behaviours to fail
+		// only when navigating through the editor screens (rather than going directly to the editor by URL).
+		// See: https://github.com/WordPress/gutenberg/pull/56856.
+		//
+		// Example (what we could do):
+		// await admin.visitSiteEditor( {
+		// 	postId: createdMenu?.id,
+		// 	postType: 'wp_navigation',
+		// } );
+		//
+		await admin.visitSiteEditor();
+
+		// create a Navigation Menu called "Test Menu" usinf the REST API helpers
+		const createdMenu = await requestUtils.createNavigationMenu( {
+			title: 'Primary Menu',
+			content:
+				'<!-- wp:navigation-link {"label":"WordPress","type":"custom","url":"http://www.wordpress.org/","kind":"custom"} /-->',
+		} );
+
+		// Add another so we get a list of Navigation menus in the editor.
+		await requestUtils.createNavigationMenu( {
+			title: 'Another One',
+			content:
+				'<!-- wp:navigation-link {"label":"Another Item","type":"custom","url":"http://www.wordpress.org/","kind":"custom"} /-->',
+		} );
+
+		const editorSidebar = page.getByRole( 'region', {
+			name: 'Navigation',
+		} );
+
+		await editorSidebar
+			.getByRole( 'button', {
+				name: 'Navigation',
+			} )
+			.click();
+
+		// Wait for list of Navigations to appear.
+		editorSidebar.getByRole( 'heading', {
+			name: 'Navigation',
+			level: 1,
+		} );
+
+		await editorSidebar
+			.getByRole( 'button', {
+				name: 'Primary Menu',
+			} )
+			.click();
+
+		await expect( page ).toHaveURL(
+			`wp-admin/site-editor.php?postId=${ createdMenu?.id }&postType=wp_navigation`
+		);
+
+		// Wait for list of Navigations to appear.
+		editorSidebar.getByRole( 'heading', {
+			name: 'Primary Menu',
+			level: 1,
+		} );
+
+		// Switch to editing the Navigation Menu
+		await editorSidebar
+			.getByRole( 'link', {
+				name: 'Edit',
+			} )
+			.click();
+
+		// Open List View.
+		await pageUtils.pressKeys( 'access+o' );
+
+		const listView = page
+			.getByRole( 'region', {
+				name: 'List View',
+			} )
+			.getByRole( 'treegrid', {
+				name: 'Block navigation structure',
+			} );
+
+		await expect( listView ).toBeVisible();
+
+		const navBlockNode = listView.getByRole( 'link', {
+			name: 'Navigation (locked)',
+			exact: true,
+		} );
+
+		// The Navigation block should be present and locked.
```

Comment:
Do we want to be this specific? If we make it unlocked in the future we'll need to update the test. Is it ok to just check that we have the block?

Reply:
That is specifically the reason for writing these tests so the answer is yes.

Reply:
If the block is unlocked then the whole UX of the focus mode is broken.